packages/flutter/test/material/chip_test.dart

```diff
@@ -1403,6 +1403,44 @@ void main() {
     expect(deleted, true);
   });
 
+  testWidgets('Chip elevation works correctly', (WidgetTester tester) async {
```

Comment:
Could you do a simplified version of https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/material_test.dart#L123

Just to check that the RenderObject gets the correct shadow?

Reply:
Looks like no :(